.github/workflow/linters.yml

```diff
@@ -0,0 +1,37 @@
+name: Linters
```

Comment:
- [ ] You've workflow added but no linter is present on the Pull Request.

Reply:
Thank you for your feedback @arslanbisharat , this will be fixed shortly.